weldx/geometry.py

```diff
@@ -2457,40 +2569,14 @@ def from_geometry_raster(
             New `SpatialData` instance
 
         """
-        # todo: this needs a test
-        # todo: workaround ... fix the real problem
-        # if not isinstance(geometry_raster, np.ndarray):
-        #    geometry_raster = np.array(geometry_raster)
-        if geometry_raster[0].ndim == 2:
-            return SpatialData(*_triangulate_geometry(geometry_raster))
```

Comment:
I removed this branch entirely. I am not sure if this is still an issue, however, notebooks and tests run without it. To my understanding, this branch should just catch cases where a single shape was passed to the profile. I tested a profile with a single shape and it passed. So I guess it is fine. If not, will find out.